templates/Coaster/show.html.twig

```diff
@@ -286,6 +294,7 @@
 {% endblock %}
 
 {% block javascripts %}
+  <script src="{{ asset('js/pages/apexcharts.min.js') }}"></script>
```

Comment:
why did you choose apex charts? (out of curiosity)

Reply:
Imo it has more features and customizations possible than chart.js. Highcharts could have been a solution as well but I wanted to try something new. UI wise I find apexcharts more interesting, less outdated and there's nice interaction and animation possibilities if we need to use charts elsewhere on the website in the future.